cmd/state/main.go

```diff
@@ -127,6 +128,11 @@ func run(args []string, isInteractive bool, out output.Outputer) error {
 	machineid.SetErrorLogger(logging.Error)
 	logging.UpdateConfig(cfg)
 
+	svcm := svcmanager.New(cfg)
+	if err := svcm.Start(); err != nil {
```

Comment:
Why don't we have to wait here for it to become ready?

Reply:
Because that would slow down every single state tool invocation. If we have a command that depends on this we can pass the `svcm` down to it and then it cal `Wait()`.